Plugins/Public/base_plugin/PlayerBase.cpp

```diff
@@ -536,14 +547,21 @@ float PlayerBase::GetAttitudeTowardsClient(uint client)
 		{
 			return 1.0;
 		}
-	}
+	}	
```

Comment:
???

Reply:
What? Changed by accident. Not sure how to remove it.

Reply:
The resulting change changes nothing anyway.